### PR TITLE
Deleting nonexisting content in English

### PR DESCRIPTION
**Pull request by Vallejoanderson**
The following content deleted does not exist in English and is a broken link in Spanish too.

**English version:**

https://developer.mozilla.org/en-US/docs/Web/JavaScript

**Spanish version:** 

https://developer.mozilla.org/es/docs/Web/JavaScript

### CODE REVIEW

**Review by JuanVqz — APPROVED** [review]

**Review by JuanVqz — CHANGES_REQUESTED** [review]
coincidir con la version en ingles

**Review by Vallejoanderson — COMMENTED** [review]
Se ha hecho coincidir mas fielmente desde la base en ingles y se ha pasado a formato md.

**Comment by JuanVqz** [comment]
> Se ha hecho coincidir mas fielmente desde la base en ingles y se ha pasado a formato md.

Gracias @Vallejoanderson pero al parecer tienes un conflicto que resolver, puedes hacerlo desde aquí mismo en github en el boton `Resolver conflictos` o en tu ambiente local, si tienes dudas de como resolverlo, contactanos en el chat

y creo que el bueno para revisar este PR es @Graywolf9

**Review by JuanVqz — APPROVED** [review]

File: files/es/web/javascript/index.md
Comment (cw118) [inline files/es/web/javascript/index.md:12]: Some external URLs/resources in English also have translated versions. If you find that they're good quality, feel free to link to the Spanish version of that resource rather than the English one :)

I only checked a couple of the Wikipedia pages, but these look like they *could* be linked to in place of the English ones (this is, of course, completely up to you):

- https://es.wikipedia.org/wiki/Java_(lenguaje_de_programaci%C3%B3n)
- https://es.wikipedia.org/wiki/JavaScript

File: files/es/web/javascript/index.md
Comment (cw118) [inline files/es/web/javascript/index.md:18]: Looks like this `(https://en.wikipedia.org/wiki/Java_(programming_language) )` has an extra space and closing parenthesis `)` at the end, in addition to the possibility of linking to the Spanish counterpart and not the English one (see comment above)

File: files/es/web/javascript/index.md
Comment (cw118) [inline files/es/web/javascript/index.md:20]: Once the new card keywords are added for the `es` locale (see [this pull request by JuanVqz at mdn/yari](https://github.com/mdn/yari/pull/5225), the **Callout:** will probably have to be changed to the **Observación:** keyword

File: files/es/web/javascript/index.md
Comment (cw118) [inline files/es/web/javascript/index.md:82]: Being a little picky, but there's an extra newline here

File: files/es/web/javascript/index.md
Comment (cw118) [inline files/es/web/javascript/index.md:33]: Being a bit picky here, but there's an extra space after `(/es/docs/Learn/JavaScript )` (in the link)

File: files/es/web/javascript/index.md
Comment (cw118) [inline files/es/web/javascript/index.md:92]: Another extra newline here **and** at line 96 (or 97, depending on which one you decide to remove)

Comment (cw118) [reply files/es/web/javascript/index.md:92]: N.B. There are some "extra" single newlines between bullet points as well (e.g. line 57), for example:

- Text here

- More text here

I didn't comment on those occurrences since they technically shouldn't change the formatting/layout of the page in the end, so it really has to do with Markdown syntax/style and consistency

**Review by cw118 — COMMENTED** [review]
Hi! I hope you don't mind if I make a few suggestions here, mainly small typos and a bit of nitpicking. At the end of the day, whether these suggestions are to be accepted is completely up to you :)

File: files/es/web/javascript/index.md
Comment (JuanVqz) [reply files/es/web/javascript/index.md:12]: it totally makes sense, I also agree, I'll share this "small but minute detail" with the rest of the community. 

I'll add the suggestion for you, why don't have permission to suggest edits?

Comment (JuanVqz) [inline files/es/web/javascript/index.md:12]:
```suggestion
**JavaScript** (**JS**) es un lenguaje de programación ligero, interpretado, o compilado [justo-a-tiempo](https://es.wikipedia.org/wiki/Compilaci%C3%B3n_en_tiempo_de_ejecuci%C3%B3n) (just-in-time) con {{Glossary("First-class Function", "funciones de primera clase")}}. Si bien es más conocido como un lenguaje de scripting (secuencias de comandos) para páginas web, y es usado en [muchos entornos fuera del navegador](https://es.wikipedia.org/wiki/JavaScript), tal como {{Glossary("Node.js")}}, [Apache CouchDB](https://couchdb.apache.org/) y [Adobe Acrobat](https://www.adobe.com/devnet/acrobat/javascript.html) JavaScript es un lenguaje de {{Glossary ("Prototype-based programming", "programación basada en prototipos")}}, multiparadigma, de un solo hilo, dinámico, con soporte para programación orientada a objetos, imperativa y declarativa (por ejemplo programación funcional). Lee más en [acerca de JavaScript](/es/docs/Web/JavaScript/About_JavaScript).
```

Comment (cw118) [reply files/es/web/javascript/index.md:12]: Oops, that was a mix-up on my part. I forgot the syntax for adding suggestions and for some reason thought that doing so required specific permissions, I'll edit them now :)

File: files/es/web/javascript/index.md
Comment (JuanVqz) [reply files/es/web/javascript/index.md:18]: No, it is not an extra `)` it is part of the wiki URL, https://es.wikipedia.org/wiki/Java_(lenguaje_de_programaci%C3%B3n)

File: files/es/web/javascript/index.md
Comment (cw118) [reply files/es/web/javascript/index.md:20]: N.B. if this is changed to the Spanish keyword before the yari PR is merged, it won't have the expected styling, but that problem should "go away" once the `es.json` file is created

Comment (JuanVqz) [reply files/es/web/javascript/index.md:20]: good to know because I thought yari will find the word `Callout` and replace it by itself.

File: files/es/web/javascript/index.md
Comment (cw118) [inline files/es/web/javascript/index.md:96]:
```suggestion
Herramientas útiles para escribir y depurar tu código **JavaScript**.

- [Herramientas para Desarrolladores de Firefox](/es/docs/Tools)
```

File: files/es/web/javascript/index.md
Comment (JuanVqz) [inline files/es/web/javascript/index.md:20]: Yo habia pensado que `Observación` era una buena traducción para `Callout` pero en este parrafo lo estoy dudando, que piensan ustedes? @mdn/yari-content-es

```suggestion
> **Observación:** **Quieres transformarte en un desarrollador web front-end?**
```

File: files/es/web/javascript/index.md
Comment (JuanVqz) [reply files/es/web/javascript/index.md:33]: good eye! :+1:

Comment (JuanVqz) [inline files/es/web/javascript/index.md:33]:
```suggestion
Dirígete a nuestro tema del [Area de Aprendizaje JavaScript](/es/docs/Learn/JavaScript) si quieres aprender JavaScript pero no tienes experiencia previa de JavaScript o programación. Los módulos completos disponibles allí son los siguientes:
```

File: files/es/web/javascript/index.md
Comment (cw118) [inline files/es/web/javascript/index.md:114]:
```suggestion
- [JSBin](https://jsbin.com/)
  - : JSBin es una herramienta de depuración de desarrollo web colaborativo de código abierto.
- [Codepen](https://codepen.io/) 
  - : Codepen es otra herramienta colaborativa de desarrollo web usada como un espacio para mostrar resultados en vivo.
```

File: files/es/web/javascript/index.md
Comment (JuanVqz) [inline files/es/web/javascript/index.md:82]:
```suggestion
```

File: files/es/web/javascript/index.md
Comment (cw118) [inline files/es/web/javascript/index.md:117]:
```suggestion
- : Codepen es otra herramienta colaborativa de desarrollo web usada como un espacio para mostrar resultados en vivo.
- [StackBlitz](https://stackblitz.com/)
  - : StackBlitz es otra herramienta de depuración/visualización en línea, que puede alojar e implementar aplicaciones full-stack usando React, Angular, etc.
```

File: files/es/web/javascript/index.md
Comment (cw118) [inline files/es/web/javascript/index.md:57]: Unnecessary newline

```suggestion
- [Una reintroducción a JavaScript](/es/docs/Web/JavaScript/A_re-introduction_to_JavaScript)
  - : Una descripción general para aquellos que _piensan_ que saben JavaScript.
- [Estructuras de datos en JavaScript](/es/docs/Web/JavaScript/Data_structures)
  - : Descripción general de las estructuras de datos disponibles en JavaScript.
```

File: files/es/web/javascript/index.md
Comment (JuanVqz) [reply files/es/web/javascript/index.md:82]: I have seen it but I decided not to say anything since now we are two, I think it's time to do it.

Do you know a good `Prettier`, `linter`, or `auto-formatted for markdown? which editor?

File: files/es/web/javascript/index.md
Comment (JuanVqz) [inline files/es/web/javascript/index.md:92]:
```suggestion
```

**Review by cw118 — COMMENTED** [review]
A few more small formatting suggestions

File: files/es/web/javascript/index.md
Comment (JuanVqz) [inline files/es/web/javascript/index.md:57]:
```suggestion
```

File: files/es/web/javascript/index.md
Comment (JuanVqz) [inline files/es/web/javascript/index.md:114]:
```suggestion
```

File: files/es/web/javascript/index.md
Comment (JuanVqz) [inline files/es/web/javascript/index.md:117]:
```suggestion
```

File: files/es/web/javascript/index.md
Comment (cw118) [reply files/es/web/javascript/index.md:18]: Got it, didn't see that 👍

**Review by JuanVqz — APPROVED** [review]
@Vallejoanderson podrias revisar los comentarios y hacer las modificaciones correspondientes, por favor?
Si tienes algún comentario, o deseas abundar igual comentalo aquí o en matrix, saludos!

y de verdad gracias, por que ya te lo hemos regresado varias veces :D
(por cierto, puedes usar el botton `commit suggestion` para que sea mas rápido agregar los cambios)

File: files/es/web/javascript/index.md
Comment (cw118) [reply files/es/web/javascript/index.md:33]: Sorry, I started converting my comments to suggestions — feel free to delete/resolve my old ones or ask me to delete mine :)

File: files/es/web/javascript/index.md
Comment (JuanVqz) [reply files/es/web/javascript/index.md:92]: @cw118 if you still helping us, you will learn Spanish :smile:

File: files/es/web/javascript/index.md
Comment (cw118) [reply files/es/web/javascript/index.md:82]: There are lots out there, I've recently been trying out [`markdownlint`](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) in VS Code

**Comment by cw118** [comment]
If I may (have to say this in English, but I just might have to learn Spanish now :)), to avoid adding lots and lots of commits through accepting suggestions, you can add them all to a batch (`Add suggestion to batch`, I believe) then commit them all in one

File: files/es/web/javascript/index.md
Comment (JuanVqz) [reply files/es/web/javascript/index.md:12]: hey @cw118 taking advantage you are here, we were wondering if we could add an ISSUE_TEMPLATE for the Spanish version, it could be challenging for Spanisk Speakers to open a new issue and see the questions in English, we are thinking in maybe add the same questionary but translated to Spanish, what do you think?

Comment (cw118) [reply files/es/web/javascript/index.md:12]: I might not be the best person to ask (maybe a question for other maintainers/review teams?), but I personally think that'd be great :) 

And just to avoid filling up this PR/thread with lots of comments, would it be alright if I join Spanish MDN matrix chat? It'd be the perfect platform for discussions like these (and now I really have to learn Spanish)

File: files/es/web/javascript/index.md
Comment (Vallejoanderson) [reply files/es/web/javascript/index.md:20]: Suena bien, aunque me preocupa que no se estilice como sucede en la version en ingles.

File: files/es/web/javascript/index.md
Comment (Vallejoanderson) [reply files/es/web/javascript/index.md]: Hi @cw118. I am new at this and I appreciate your help. Thank you for helping us to have a more presentable PR (I hope I have expressed myself well) , I can't be more agree about your proposal to use links in Spanish instead of the English one.

Comment (cw118) [reply files/es/web/javascript/index.md]: **Just some important stuff to add to the external links detail**: 

- Some resources, including official specifications, are only available in English (e.g. `https://tc39.es/ecma402/`), so localizers just use the English resource
- Even with others where translations *are* available, they might not be "good enough quality" to include — this is up to the community (contributors and reviewers) to verify — in which case it could be better to stick with the original English link. 

I hope I explained that okay. Either way, if it's alright with everyone, I think I'm going to join the `l10n-es` Matrix chat and we could discuss everything further there?

> I am new at this and I appreciate your help. Thank you for helping us to have a more presentable PR (I hope I have expressed myself well)

Not a problem, it's so nice to see the Spanish community back in action here! You guys are doing great :)

File: files/es/web/javascript/index.md
Comment (Graywolf9) [inline files/es/web/javascript/index.md:95]: Linea repetida

File: files/es/web/javascript/index.md
Comment (Graywolf9) [inline files/es/web/javascript/index.md:70]: Verificar traducción `typed arrays` (`arreglos escritos`)

File: files/es/web/javascript/index.md
Comment (Graywolf9) [inline files/es/web/javascript/index.md:81]: typo en "Objectos"

File: files/es/web/javascript/index.md
Comment (Graywolf9) [inline files/es/web/javascript/index.md:92]: línea repetida

File: files/es/web/javascript/index.md
Comment (Graywolf9) [inline files/es/web/javascript/index.md:99]: Verificar `shell` (`consola`)

File: files/es/web/javascript/index.md
Comment (Graywolf9) [inline files/es/web/javascript/index.md:104]: Revisar congruencia de la traducción

File: files/es/web/javascript/index.md
Comment (Vallejoanderson) [reply files/es/web/javascript/index.md:95]: Listo, borrada.

File: files/es/web/javascript/index.md
Comment (Vallejoanderson) [reply files/es/web/javascript/index.md:70]: Lo he sustituido por arreglos tipados.

File: files/es/web/javascript/index.md
Comment (Vallejoanderson) [reply files/es/web/javascript/index.md:98]: Modificado.

File: files/es/web/javascript/index.md
Comment (Vallejoanderson) [reply files/es/web/javascript/index.md]: Listo @Graywolf9, ya hice todos los cambios espero no se me haya pasado ninguno, me avisas si tienes otra observacion, por fa. Gracias por la ayuda.

File: files/es/web/javascript/index.md
Comment (JuanVqz) [inline files/es/web/javascript/index.md:18]: Como nos sugería cw118, de ser posible, estaria bueno cambiar los enlaces a referencias externas en Español (equivalente a la referencia en Inglés) 

```suggestion
No confundas JavaScript con el [lenguaje de programación Java](https://es.wikipedia.org/wiki/Java_(lenguaje_de_programaci%C3%B3n)). Ambos "Java" y "JavaScript" son marcas o marcas registradas de Oracle en los Estados Unidos y otros países, Sin embargo, los dos lenguajes de programación tienen sintaxis, semántica y usos muy diferentes.
```

File: files/es/web/javascript/index.md
Comment (JuanVqz) [reply files/es/web/javascript/index.md:92]: @Vallejoanderson aquí no veo el cambio, si lo hiciste?

**Review by JuanVqz — APPROVED** [review]

File: files/es/web/javascript/index.md
Comment (Vallejoanderson) [reply files/es/web/javascript/index.md:92]: Sip, fue borrada :D

File: files/es/web/javascript/index.md
Comment (Vallejoanderson) [reply files/es/web/javascript/index.md]: Listo, @JuanVqz ya lo modifique y le di una revisada por si quedaba otro link en ingles.

**Comment by JuanVqz** [comment]
@Vallejoanderson gracias por tu contribución, también gracias a los que revisaron @cw118 @Graywolf9 :t-rex:

**Pull request merged**
